build.py

```diff
@@ -124,9 +124,30 @@ def run(config, start_class_path, arguments):
 ### def run ###
 
 
+# USAGE
+#
+# Print basic usage info.
+#
+def usage(args) :
+  print('Unknown command "%s"' % args)
```

Comment:
I don't like having usage print unknown command. Can we have usage take no arguments and just print the usage info? There is only one case where "Unknown command" needs to be printed but multiple cases when usage data needs to be printed.